# src/users/domain/exceptions.py
class UserNotFoundException(Exception):
    pass